### PR TITLE
Add org.enjoyingfoss.FOSStriangulator

### PR DESCRIPTION
**Pull request by 12people**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project.
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

Just a note about permissions: I'm allowing access to the user's home folder rather than using a portal. This app is an editor, so the user needs to be able to access their home folder. It's also a Processing app (= Java app) and uses Java dialogs. It seems to me that replacing these with portals would be a daunting task if not downright impossible.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build org.enjoyingfoss.FOSStriangulator

File: org.enjoyingfoss.FOSStriangulator.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        sha256: 6c95f19006a770e957754b5afc9f8624a04142eb87e5cdf78017cb6b5729309e
+        url: https://github.com/FOSStriangulator/FOSStriangulator/releases/download/v1.0.0/FOSStriangulator.tar.xz
+      - type: file
+        path: org.enjoyingfoss.FOSStriangulator.desktop
+      - type: file
+        path: org.enjoyingfoss.FOSStriangulator.metainfo.xml
+      - type: file
+        path: org.enjoyingfoss.FOSStriangulator.svg
+      - type: file
+        path: org.enjoyingfoss.FOSStriangulator-symbolic.svg
```
Comment (barthalion) [inline org.enjoyingfoss.FOSStriangulator.yml:36]: Ideally these files should be part of the `tar.xz` above and maintained in https://github.com/FOSStriangulator/FOSStriangulator

Comment (12people) [reply org.enjoyingfoss.FOSStriangulator.yml]: @barthalion Should be fixed with the latest commit

**Comment by barthalion** [comment]
bot, build org.enjoyingfoss.FOSStriangulator

**Pull request closed without merging**